### PR TITLE
add better link for gcp auth plugin doc

### LINKED ISSUE #103499: Update gcp auth plugin deprecation with actionable message
We should drop the `.V(1)` bit once in favor of a `klog.Warningf` once gcp has a replacement.

xref: https://github.com/kubernetes/kubernetes/pull/102181#discussion_r636963736

/assign @enj @mikedanese 
/milestone v1.24
/triage accepted
/sig auth

### PR DESCRIPTION
**Pull request by mikedanese**
/kind documentation
/sig auth

Fixes #103499

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func newGCPAuthProvider(_ string, gcpConfig map[string]string, persister restcli
 	// this should be updated to use klog.Warningf in v1.24 to more actively warn consumers
 	warnOnce.Do(func() {
 		klog.V(1).Infof(`WARNING: the gcp auth plugin is deprecated in v1.22+, unavailable in v1.25+; use gcloud instead.
```
Comment (liggitt) [inline staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp.go:122]: switch to klog.Warningf as well here and remove the todo above?

Comment (mikedanese) [reply staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp.go]: Done.

**Pull request merged**